docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+   - [Translation modules](#using-translation-modules)
+3. Remove redundant code, using [tree shaking](#tree-shaking).
+
+### Importing required modules
+
+No matter which of the [optional modules](#list-of-all-modules) you use, the following modules are always required:
+
+- Handsontable's core functionalities
+- The default [cell type](@/guides/cell-types/cell-type.md): `text`
+- [moment.js](https://momentjs.com/)
+- [DOMpurify](https://www.npmjs.com/package/dompurify)
+- [core-js](https://www.npmjs.com/package/core-js)
+- Handsontable's CSS
+
+To import the required modules, follow the steps below.
+
+#### Step 1: Remove the full Handsontable import
+
+When using modules, you no longer need to import the full version of Handsontable.
+
+Remove the `handsontable` import:
+
+```js
+// remove this line
+import Handsontable from 'handsontable';
+```
+
+#### Step 2: Import the `handsontable/base` module
+
+Instead of the full Handsontable version, import the `handsontable/base` module.
 
-To use a specific editor you need to import it alongside the registering method. For instance, let's try adding the password editor.
+```js
+// import the `handsontable/base` module
+import Handsontable from 'handsontable/base';
+```
+
+#### Step 3: Import the CSS
+
+Handsontable's CSS file is not modularized, so you need to import it:
+
+```js
+import Handsontable from 'handsontable/base';
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
+```
+
+Now, you're ready to use [any optional modules](#list-of-all-modules) of your choice.
+
+### Using editor modules
+
+Each editor module contains a different type of [cell editor](@/guides/cell-functions/cell-editor.md):
 
-Start with importing the base, `PasswordEditor` and the `registerEditor` method.
+| Module               | Cell editor alias |
+| -------------------- | ----------------- |
+| `AutocompleteEditor` | `autocomplete`    |
+| `BaseEditor`         | `base`            |
+| `CheckboxEditor`     | `checkbox`        |
+| `DateEditor`         | `date`            |
+| `DropdownEditor`     | `dropdown`        |
+| `HandsontableEditor` | `handsontable`    |
+| `NumericEditor`      | `numeric`         |
+| `PasswordEditor`     | `password`        |
+| `SelectEditor`       | `select`          |
+| `TextEditor`         | `text`            |
+
+To start using an editor module, follow the steps below.
+
+#### Step 1: Import the required modules
+
+First, make sure to [import the required modules](#importing-required-modules):
 
 ```js
-// remember to have the base imported
+// import the `handsontable/base` module
 import Handsontable from 'handsontable/base';
-// import the method for registering an editor
-// import the PasswordEditor
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
+```
+
+#### Step 2: Import the editors' registering method
+
+Import the `registerEditor()` method that lets you register your chosen editor:
```

Comment:
This is a pleonasm. I think that we owe the reader any explanation to better understand what's going on:

> Handsontable will only recognize the editor module that you've imported, if you provide it to a registering function.

Also, I think that there are too many steps. See my comment on line 12 of `vue-modules.md`.

Reply:
Not sure I understand, I don't see a pleonasm in this structure:

> Import the `[name]` function that lets you do [sth].

Just because a function/method name **seems** self-explanatory, doesn't mean that it is: that function could do sth entirely else than the name suggests (as sometimes is the case in HoT).

How about:

> Import the `registerEditor()` function that lets Handsontable recognize your imported editor module.

Reply:
> Just because a function/method name *seems* self-explanatory, doesn't mean that it is: that function could do sth entirely else than the name suggests (as sometimes is the case in HoT).

Can't argue with that, but still I think that a sentence "the `registerEditor()` function registers a chosen editor" make the reader roll their eyes. Your proposed version sounds much better.